app/views/publications/mid_cycle_report/show.html.erb

```diff
@@ -0,0 +1,213 @@
+              </a>
+            </li>
+          </ol>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m" id="data">1. How this data compares to other data we publish</h2>
+      <p class="govuk-body">The page showing <%= govuk_link_to('monthly statistics for courses starting in the 2023 to 2024 academic year', publications_monthly_report_itt_path(RecruitmentCycle.current_year)) %>, calculates data using the same method UCAS used to. This method counts a candidate’s furthest application. For example, if an application moves from ‘recruited’ to ‘deferred’ the recruited count will go down, and the deferred count will go up. This method does not easily show where a candidate ends up over time.</p>
+      <p class="govuk-body">This page about national recruitment performance, calculates data in a new way by counting all submitted applications, regardless of their outcome. It also counts all offers regardless of if the offer is accepted, declined or waiting for a decision.</p>
+      <p class="govuk-body">With each page using a different method, you should not compare this page with the monthly statistics page.</p>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m" id="submitted">2. Candidates who’ve submitted applications</h2>
+      <p class="govuk-body">The number of candidates who’ve submitted at least one application to an ITT provider between 11 October 2022 to 3 April 2023.</p>
+    </div>
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <%= render 'table', model: @national_data, first_column: 'Candidates submitted', second_column: 'Candidates submitted (previous cycle)', third_column: 'Candidates submitted change', include_total: false, top_level_only: true, subject_table: false %>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m" id="number-applications">3. Number of applications</h2>
+      <p class="govuk-body">The total number of applications received by all ITT providers from 11 October 2022 to 3 April 2023.</p>
+    </div>
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <%= render 'table', model: @national_data, first_column: 'Applications submitted (this cycle)', second_column: 'Applications submitted (previous cycle)', third_column: 'Applications submitted change', include_total: true, top_level_only: false, subject_table: true %>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m" id="offer">4. Applications with an offer</h2>
+      <p class="govuk-body">The number of applications with an offer from an ITT training provider from 11 October 2022 to 3 April 2023.</p>
```

Comment:
**Replace content with:**
The number of applications with an offer made to a candidate between 11 October 2022 and 3 April 2023.  

The table includes applications: 

- with offers withdrawn by the provider 
- withdrawn by candidates 
- deferred until next cycle 

The table does not include deferrals from a previous cycle and then reconfirmed by the provider this cycle.